docs/installation/installing-with-npm.md

```diff
@@ -283,7 +283,7 @@ To use different asset paths, also complete the below step(s).
   @import "govuk-frontend/all";
```

Comment:
Nice work so far, I think we need to make sure the imports now have the directory too?